electrum/gui/kivy/uix/dialogs/cpfp_dialog.py

```diff
@@ -0,0 +1,161 @@
+from typing import TYPE_CHECKING, Optional
+
+from kivy.app import App
+from kivy.factory import Factory
+from kivy.properties import ObjectProperty
+from kivy.lang import Builder
+
+from electrum.gui.kivy.i18n import _
+
+if TYPE_CHECKING:
+    from ...main_window import ElectrumWindow
+
+from .fee_dialog import FeeSliderDialog
+from electrum.gui.qt.amountedit import BTCAmountEdit
+from electrum.util import format_satoshis_plain
+
+Builder.load_string('''
+<CPFPDialog@Popup>
+    title: _('CPFP')
+    size_hint: 0.8, 0.8
+    pos_hint: {'top':0.9}
+    method: 0
+    BoxLayout:
+        orientation: 'vertical'
+        padding: '10dp'
+        GridLayout:
+            height: self.minimum_height
+            size_hint_y: None
+            cols: 1
+            spacing: '10dp'
+            BoxLabel:
+                id: msg1
+                text: _('CPFP stands for Child Pays for Parent')
+            BoxLabel:
+                id: total_size
+                text: _('Total Size')
+                value: ''
+            BoxLabel:
+                id: input_amount
+                text: _('Input amount')
+                value: ''
+            BoxLabel:
+                id: output_amount
+                text: _('Output amount')
+                value: ''
+            BoxLabel:
+                id: fee_for_child
+                text: _('Fee for child')
+                value: ''
+        BoxLayout:
+            orientation: 'horizontal'
+            size_hint: 1, 0.5
+            Label:
+                text: _('Target') + ' (%s):' % (_('mempool') if root.method == 2 else _('ETA') if root.method == 1 else _('static'))
+            Button:
+                id: fee_target
+                text: ''
+                background_color: (0,0,0,0)
+                bold: True
+                on_release:
+                    root.method  = (root.method + 1) % 3
+                    root.update_slider()
+                    root.on_slider(root.slider.value)
+
+        Slider:
+            id: slider
+            range: 0, 4
+            step: 1
+            on_value: root.on_slider(self.value)
+        BoxLayout:
+            orientation: 'horizontal'
+            size_hint: 1, 0.2
+            Label:
+                text: _('Final')
+            CheckBox:
+                id: final_cb
```

Comment:
Let's just remove the "final" checkbox. You are not even using the value of the checkbox atm. :P
(The Qt GUI does not have a checkbox either. It's ok to always opt-in to RBF for CPFP txs, and this is what happens already.)

Make sure to also remove related code, e.g. its value is passed to the callback currently.

```suggestion
```